src/Automation/Automation/Common/AutomationPSClientSoftwareUpdateConfiguration.cs

```diff
@@ -79,7 +79,7 @@ public SoftwareUpdateConfiguration CreateSoftwareUpdateConfiguration(string reso
 
                 var sucParameters = new Sdk.SoftwareUpdateConfiguration()
                 {
-                    ScheduleInfo = new Sdk.ScheduleProperties()
+                    ScheduleInfo = new Sdk.SUCScheduleProperties()
```

Comment:
What's the reason of this change?

Reply:
ScheduleProperties was updated to SUCScheduleProperties in the swagger definition for Software Update Configuration schedule properties, hence this change